utils/trivia.py

```diff
@@ -96,10 +96,10 @@ def end(self) -> int:
     def check(self, answer) -> int:
         # Return non-zero value to indicate failure
         # Return 0 to indicate success
-        if (self.qAsked.answer == answer):
-            return 0
-        else:
-            return -1
+        #if (self.qAsked.answer == answer)
```

Comment:
I would try avoid committing commented out code as much as possible. A couple years ago I had to work on a project with a couple guys who did this a lot. There code became extremely hard to read and while trying to read through their code base to figure out how to develop a feature for it, I spent around 3-4 hours testing and tracing through code paths only to find out they did nothing because the code that would've used them was completely commented out. They had replaced an entire section of the code base (several files) and hadn't removed the obsolete files. Large portions of code was also commented out in still relevant files.

This was on a government sponsored project that was expected to be delivered to the customer. They were eventually told that they had to remove all that stuff before they would be allowed to move forward with it.

Reply:
Ah, thanks for pointing that out, I had forgotten I left a comment in there.